Code2pdf/code2pdf.py

```diff
@@ -142,9 +147,19 @@ def parse_arg():
 
 def main():
     args = parse_arg()
-    pdf_file = get_output_file(args.filename, args.outputfile)
-    pdf = Code2pdf(args.filename, pdf_file, args.size)
-    pdf.init_print(linenos=args.linenos, style=args.style)
+    if  (args.filelist == False):
+        pdf_file = get_output_file(args.filename, args.outputfile)
+        pdf = Code2pdf(args.filename, pdf_file, args.size)
+        pdf.init_print(linenos=args.linenos, style=args.style)
+    else:
```

Comment:
@cibinjoseph Hi! Can we add a test suite for your changes before we merge? Also, update the usage/argparser instructions accordingly. Thanks! :)

Reply:
(On the test suite) Sure! How do you wanna go about it?
And I had already updated the argparser instructions in the commit.

Reply:
I'll edit the 'usage' section in a while!

Reply:
@cibinjoseph You may find some test suit written under `tests/test.py`. You may add yours there as well.

Reply:
Okay, got it!